src/examples/rpp/doxygen/reduce.cpp

```diff
@@ -9,17 +12,17 @@
 int main()
 {
     //! [reduce]
-    rpp::source::just(1, 2, 3)
-        | rpp::operators::reduce(10, std::plus<int>{})
-        | rpp::operators::subscribe([](int v) { std::cout << v << std::endl; });
-    // Output: 16
+    rpp::source::just("b"s, "c"s)
+        | rpp::operators::reduce("a"s, std::plus<std::string>{})
```

Comment:
No, my original idea was to make reduce working with different types. Like getting observalbe of ints and then reducing it to concatenated std::string =)

Reply:
But NVM =)